slices/xz-utils.yaml

```diff
@@ -0,0 +1,36 @@
+package: xz-utils
+
+essential:
+  - xz-utils_copyright
+
+slices:
+  bins:
```

Comment:
this should either depend on the shell scripts the package brings as they are executables as well, or we should introduce an `execs` slice that bundles all of them imo

Reply:
Using the `strings` util, it does not look like these bins reference the scripts. That seems to fix expectations as it looks like the scripts extend common utils using the binaries. 

Here is a sample from xzgrep:
```
# xzgrep -- a wrapper around a grep program that decompresses files as needed
# Instead of unsetting XZ_OPT, just make sure that xz will use file format
xz='xz --format=auto'

```